test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -725,22 +725,22 @@
       measurePods: 500
   - name: 5000Nodes
     labels: [performance]
-    threshold: 120
+    threshold: 250
```

Comment:
I think 250 is too risky. Let's change it to 200.

Reply:
I was about to wait for CI results to get better numbers, so here they are:
Unschedulable/5kNodes/10kPods - 259.5 SchedulingThroughput/Average
PreemptionAsync/5000Nodes - 271.6 SchedulingThroughput/Average

Do you think 200 is the right threshold, or something like 220 for both?

Reply:
I would set 200 for both. CI sometimes has performance drops where throughput is noticeably lower. Given that these drops are more or less periodic, the performance drop will still be noticeable, even if the threshold is quite pessimistic.